packages/moonstone/IncrementSlider/IncrementSlider.js

```diff
@@ -141,15 +140,15 @@ const IncrementSliderBase = kind({
 		decrementIcon: PropTypes.string,
 
 		/**
-		 * When `true`, the component is shown as disabled and does not generate events
+		 * Disables IncrementSlider and does not generate events.
```

Comment:
Disables the slider and prevents events from firing.  ?  There are a couple like this and they read a bit awkwardly as they are.  Other suggestions appreciated.